{
    "type": "bundle",
    "id": "bundle--82c28a7e-f31c-432b-8553-68be2a34b090",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--77c1b78e-f64f-42c4-bc0b-f3b6ce5ddb5d",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-05-22T00:58:03.00462Z",
            "modified": "2024-05-22T00:58:03.00462Z",
            "name": "CVE-2020-36788",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\ndrm/nouveau: avoid a use-after-free when BO init fails\n\nnouveau_bo_init() is backed by ttm_bo_init() and ferries its return code\nback to the caller. On failures, ttm_bo_init() invokes the provided\ndestructor which should de-initialize and free the memory.\n\nThus, when nouveau_bo_init() returns an error the gem object has already\nbeen released and the memory freed by nouveau_bo_del_ttm().",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2020-36788"
                }
            ]
        }
    ]
}